source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -114,9 +114,12 @@
     }
 
     EvaluationViewModel.prototype.isValueEditable = function (calification) {
-        return !this.evaluation.finished && ((((this.userLogged == calification.calificationColumn.evaluatorEmployee) ||
-            (this.userView == 3 && calification.calificationColumn.evaluatorEmployee == "_company")) && !calification.calificationColumn.finished) ||
-            this.evaluation.readyForDevolution && this.userView == 3 && (calification.calificationColumn.owner == 3 || calification.calificationColumn.owner == 0))
+        return !this.evaluation.finished
+            && ((this.evaluation.readyForDevolution && this.userView == 3
+            && (calification.calificationColumn.owner == 3 || calification.calificationColumn.owner == 0))
+            || (((this.userLogged == calification.calificationColumn.evaluatorEmployee)
+            || (this.userView == 3 && calification.calificationColumn.evaluatorEmployee == "_company"))
+            && !calification.calificationColumn.finished))
     }
```

Comment:
Regarding the numbers

``` javascript
this.userView == 3
```

Maybe it easier to read if we have some constants with meaningful names like (constants do not exist in javascript but maybe it can be simulated with an object or simple vars).